pkg/shell/superuser.jsx

```diff
@@ -108,7 +110,16 @@
 
     function init() {
         return proxy.Stop().always(() => {
-            start("sudo");
+            if (proxy.Methods) {
+                const ids = Object.keys(proxy.Methods);
+                if (ids.length == 1)
+                    start(ids[0]);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17536-20220712-071722-4c262304-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)